src/twisted/python/_setup.py

```diff
@@ -288,6 +288,7 @@ class my_build_ext(build_ext_twisted):
         "hyperlink >= 17.1.1",
         "PyHamcrest >= 1.9.0",
         "attrs >= 17.4.0",
+        "ipaddress; python_version < '3.0'",
```

Comment:
Thanks for correctly updating the package metadata for this requirement.  It strikes me as slightly unfortunate that this is our first `ipaddress` import anywhere, since we consistently use `twisted.internet.address` everywhere else.  Since I'm sending this back to you for feedback, maybe *consider* adding the IPv4 mapped test to our own class?  I wouldn't say it should be a hard requirement, since we probably transitively depend on this six different ways anyway now, and it is in the stdlib, but it does stick out just a little.

If you consider it and think it's the best choice, then by all means leave it :)

Reply:
> So I prefer to leave this as it is, although I've dropped the setup change since `python_version < '3.0'` is now always true for Twisted.

always False for Twisted, I hope :)